src/teleop_twist_joy.cpp

```diff
@@ -143,6 +147,9 @@ TeleopTwistJoy::TeleopTwistJoy(const rclcpp::NodeOptions & options)
   ROS_INFO_COND_NAMED(
     pimpl_->enable_turbo_button >= 0, "TeleopTwistJoy",
     "Turbo on button %" PRId64 ".", pimpl_->enable_turbo_button);
+  ROS_INFO_COND_NAMED(
+    1, "TeleopTwistJoy",
+    "Teleop enable inverted reverse %d.", pimpl_->inverted_reverse);
```

Comment:
I think we should only print this if the inverted_reverse is active, so:

```suggestion
  ROS_INFO_COND_NAMED(
    pimpl_->inverted_reverse, "TeleopTwistJoy", "%s", "Teleop enable inverted reverse.");
```